### ISSUE TITLE
dependency issue with reanimated 2.4 on EXPO

### ISSUE THREAD

**Issue by krskrs** [howljs/react-native-calendar-kit#83]
Hi,
we're using your library in an expo project, but after they upgraded react-reanimated to the 3.0 version, we're no longer able to use your library, because there is a dependency conflict between reanimated 2.4.x used by the library and the new 3.0 version used by expo.

Since they should be api compatible, can you please try to upgrade the reanimated requirement, or maybe just make it more loose so that the 3.0 version is also accepted?

Thank you!

**Comment by howljs** [comment]
I'm writing a new version, it will take some time to finish :'(

https://github.com/howljs/react-native-calendar-kit/assets/33460888/6b916c2a-9396-4064-b0f5-c83842bd32e4

**Comment by johnrevs** [comment]
hello there, when are you planning to release the new version?

**Comment by SpaceDux** [comment]
Any further news on this?

**Comment by joaobsantos** [comment]
@howljs if you need help contributing I'll try to help you on more simple things that you need, I noticed that you already have pinch to zoom, thats a thing I cannot get working well, even in the older version...

**Comment by arun-saleth** [comment]
@howljs when you are released new version?

**Comment by marko-mlinarevic** [comment]
same issue here

**Comment by kota113** [comment]
For a temporary fix, you can configure package.json by adding "overrides" to work around with the dependency issues. 

You can configure it like this.
package.json:
```json
"dependencies": {
    "@howljs/calendar-kit": "^1.13.2",
    ...
    "react-native-reanimated": "~3.6.3",
    "typescript": "^5.3.3"
  },
"overrides": {
    "@howljs/calendar-kit": {
      "react-native-reanimated": "$react-native-reanimated"
    }
  },
 ```

**Comment by howljs** [comment]
Fixed in v2.0.0
